.gitignore

```diff
@@ -0,0 +1,55 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Place `.gitignore` lines that are about your setup in your `.gitignore_global`; place stuff that's essential to the project in its `.gitignore`.